src/components/Layout.tsx

```diff
@@ -105,6 +130,11 @@ export function Layout() {
         </select>
       </div>
 
+      <div className="sort-row">
+        <input type="submit" value="Filter rules" onClick={() => setShowRules(!showRules)} />
+        { showRules ? <Rules state={matchOption} setState={setMatchOption} /> : null }
+      </div>
```

Comment:
It would be nice if we could show like `x results for filters` up here

Reply:
Added, but I'll wait until I add option persistence before marking this conversation resolved